terragrunt/aws/lambda.tf

```diff
@@ -0,0 +1,22 @@
+resource "aws_lambda_function" "web" {
```

Comment:
I like that you put this all together from the resources.  I suspect you know about the `lambda` module we have, but this is definitely a good way to learn how the pieces go together.

Reply:
Yeah, actually, I haven't thought about the module, lol. It's a reflex I need to develop, but I agree it's good to learn what happens under the hood.